Lint step fails on Azure CI

Involves robot (rflint) and prettier issues. @bollwyvl would you have any insight?

By the way, it is a sad day on CI, everything failing on some edge cases (RTD has a conda/pypi/preference issues with sphinx - affecting multiple users, Travis had a hiccup on Python 3.5 with pluggy/pyls weirdness - now fixed but then it has a large backlog of Mac builds so it did not report yet and now Azure...).

Relevant traceback:
```
Success: no issues found in 38 source files
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/jupyterlab-lsp/bin/rflint", line 6, in <module>
    from rflint.__main__ import main
  File "/usr/share/miniconda/envs/jupyterlab-lsp/lib/python3.7/site-packages/rflint/parser/__init__.py", line 1, in <module>
    from .parser import (SuiteFolder, ResourceFile, SuiteFile, RobotFactory, 
  File "/usr/share/miniconda/envs/jupyterlab-lsp/lib/python3.7/site-packages/rflint/parser/parser.py", line 24, in <module>
    from robot.parsing.txtreader import TxtReader
ModuleNotFoundError: No module named 'robot.parsing.txtreader'
attempt 1 of 4...
Robot Arguments
 robot --name Linux37 --outputdir /home/vsts/work/1/s/atest/output/dry_run_linux_37_1 --output /home/vsts/work/1/s/atest/output/dry_run_linux_37_1.robot.xml --log /home/vsts/work/1/s/atest/output/dry_run_linux_37_1.log.html --report /home/vsts/work/1/s/atest/output/dry_run_linux_37_1.report.html --xunit /home/vsts/work/1/s/atest/output/dry_run_linux_37_1.xunit.xml --variable OS:Linux --variable PY:37 --randomize all --dryrun --console quiet /home/vsts/work/1/s/atest
0 errors in 3 seconds
.internal/modules/cjs/loader.js:985
  throw err;
  ^

Error: Cannot find module '/home/vsts/work/1/s/node_modules/.bin/prettier'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
internal/modules/cjs/loader.js:985
  throw err;
  ^

```

--- a/ci/env-test.yml.in
+++ b/ci/env-test.yml.in
@@ -30,3 +30,4 @@ dependencies:
   - firefox
   - geckodriver
   - robotframework-seleniumlibrary
+  - robotframework <3.2

--- a/ci/job.lint.yml
+++ b/ci/job.lint.yml
@@ -34,11 +34,11 @@ jobs:
               - script: ${{ platform.activate }} jupyterlab-lsp && python scripts/integrity.py
                 displayName: check integrity of package versions
 
-              - script: ${{ platform.activate }} jupyterlab-lsp && python scripts/lint.py
-                displayName: lint backend
-
               - script: ${{ platform.activate }} jupyterlab-lsp && jlpm || jlpm || jlpm
                 displayName: install npm dependencies
+
+              - script: ${{ platform.activate }} jupyterlab-lsp && python scripts/lint.py
+                displayName: lint backend
 
               - script: ${{ platform.activate }} jupyterlab-lsp && jlpm build:schema
                 displayName: build schema so linting can complete

--- a/requirements/atest.txt
+++ b/requirements/atest.txt
@@ -1,3 +1,4 @@
 # acceptance test dependencies for jupyter_lsp
 -r ./lab.txt
 robotframework-seleniumlibrary
+robotframework <3.2

--- a/requirements/atest.yml
+++ b/requirements/atest.yml
@@ -6,5 +6,6 @@ channels:
 
 dependencies:
   - robotframework-seleniumlibrary
+  - robotframework <3.2
   - firefox
   - geckodriver

--- a/requirements/lint.yml
+++ b/requirements/lint.yml
@@ -9,3 +9,4 @@ dependencies:
   - isort
   - mypy
   - robotframework-lint
+  - robotframework <3.2

